<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MonitorView</class>
 <widget class="QWidget" name="MonitorView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>860</width>
    <height>550</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout" rowstretch="0,1,0,0,0,0,0" columnstretch="2,0,0,0">
     <item row="2" column="3">
      <widget class="QPushButton" name="pbClearLog">
       <property name="text">
        <string>Clear</string>
       </property>
      </widget>
     </item>
     <item row="5" column="3">
      <widget class="QLineEdit" name="leRunCtrlAddr">
       <property name="text">
        <string>tcp://192.168.130.74</string>
       </property>
      </widget>
     </item>
     <item row="4" column="3">
      <widget class="QPushButton" name="pbTest">
       <property name="text">
        <string>test</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="6" column="3">
      <widget class="QPushButton" name="pbConnect">
       <property name="text">
        <string>Connect</string>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QTextBrowser" name="tbLog"/>
     </item>
     <item row="3" column="3">
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="0" rowspan="4" colspan="2">
      <widget class="QGraphicsView" name="gvSensor"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
